apps/docs/src/guide/encoding/encode-and-decode.md

```diff
@@ -0,0 +1,21 @@
+# Encode and Decode
+
+Encoding is the process of transforming raw data into bytecode as per a set of rules. Decoding is the reverse process, transforming bytecode back to raw data. The SDK provides the `AbiCoder` class to encode and decode data.
+
+It has 3 static methods: `encode`, `decode`, and `getCoder`, `encode` and `decode` describe the aforementioned processes, while `getCoder` returns an instance of the internal coder required to serialize the passed type, and is used internally by the `encode` and `decode` methods. All methods expect you to pass the [ABI](https://docs.fuel.network/docs/specs/abi/json-abi-format/) and ABI Argument as function parameters, in order to deduce the specific type coders that will be required to parse the data.
```

Comment:
I was wondering if maybe more spacing could be beneficial.

```suggestion
It has three static methods:
 - `encode`
 - `decode`
 - `getCoder`

The methods `encode` and `decode` describe the aforementioned processes, while `getCoder` returns an instance of the internal coder required to serialize the passed type. This coder is then used internally by the `encode` and `decode` methods.

All methods expect you to pass the [ABI](https://docs.fuel.network/docs/specs/abi/json-abi-format/) and ABI Argument as function parameters to deduce the specific type coders that will be required to parse the data.
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2405/commits/afc7691126b4867187408fa2616a6f2bc56b1fb1